geo-replication/syncdaemon/syncdutils.py

```diff
@@ -850,8 +850,13 @@ def errlog(self):
         def logerr(l):
             logging.error(self.args[0] + "> " + l)
         for l in self.elines:
-            ls = l.split('\n')
-            ls[0] = lp + ls[0]
+            #handle py2py3 compatibility
```

Comment:
Try to fix where lines are added to `elines`(Ex: Line 888 `elines.append(b.decode())` (Line 893 with new changes))

Python3 expects bytes literal if the line is bytes literal. For example: Both `b"A:B".split(b":")` and `"A:B".split(":")` are valid in Python3

Also bytes literal is added to Python 2.6 onwards. It does nothing but the syntax is supported. `b"A:B".split(b":")` works in >Python 2.6